src/compass/utils/lut.py

```diff
@@ -23,7 +30,19 @@ def compute_geocoding_correction_luts(burst, rg_step=200, az_step=0.25):
     az_bistatic = burst.bistatic_delay(range_step=rg_step, az_step=az_step)
     rg_doppler = burst.geometrical_and_steering_doppler(range_step=rg_step, az_step=az_step)
 
+    if path_dem is None:
+        # Skip the FM mismatch rate calculation
+        az_fm_mismatch = None
+    else:
+        az_fm_mismatch = burst.az_fm_rate_mismatch_mitigation(path_dem,
+                                                              path_scratch,
+                                                              rg_step=rg_step,
+                                                              az_step=az_step)
+
     rg_lut = rg_doppler
     az_lut = az_bistatic.data
 
-    return rg_lut, az_lut
\ No newline at end of file
+    if not az_fm_mismatch is None:
+        az_lut += az_fm_mismatch.data
```

Comment:
The types don't match to perform the `+=`. I think the following will get the correct result.
```suggestion
        az_lut_data = az_bistatic.data + az_fm_mismatch.data
        az_lut = isce3.core.LUT2d(az_bistatic.x_start, az_bistatic.y_start, 
                az_bistatic.x_spacing, az_bistatic.x_spacing, az_lut_data)
```